python/shark_turbine/runtime/op_reg/compiler.py

```diff
@@ -87,10 +91,11 @@ def compile_standalone_kernel(
     start = default_timer()
     config = KernelCompileConfig(cache_key, list(device.compile_target_flags))
     kb = FreeFuncKernelBuilder.create_module(ksel, func_name=func_name)
-    ksel.op.generate(ksel, kb)
+    with kb.ip, Location.unknown():
```

Comment:
I've been putting off writing the stack extraction utility for deriving it. Naive implementations tend to make things worse because they just point to some internal bit of framework detritus and don't interop with error reporting (where you want something like "error happened here but was caused by stuff compiled from here". I know how to do it/have done it before. Just putting off. UnknownLoc is easy to find when it is time to upgrade.